content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -213,6 +213,10 @@ traffic to the internet.
 
 [Knitter](https://github.com/ZTE/Knitter/) is a network solution which supports multiple networking in Kubernetes. It provides the ability of tenant management and network management. Knitter includes a set of end-to-end NFV container networking solutions besides multiple network planes, such as keeping IP address for applications, IP address migration, etc.
 
+### Kube-OVN
+
+[Kube-OVN](https://github.com/alauda/kube-ovn) is an OVN-based kubernetes network fabric for enterprises. With the help of OVN/OVS, it provide some advance overlay network features like subnet, QoS, static IP allocation, traffic mirroring, gateway, openflow-based network policy and service proxy, etc.
```

Comment:
Change "provide" to "provides".

Don't need ", etc" at end. Suggest removing it.

Reply:
done